pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/ClustersBase.java

```diff
@@ -99,7 +98,7 @@ public void getClusters(@Suspended AsyncResponse asyncResponse) {
     @Path("/{cluster}")
     @ApiOperation(
         value = "Get the configuration for the specified cluster.",
-        response = ClusterDataImpl.class,
+        response = ClusterData.class,
```

Comment:
It does not seem a bug. Is there an error when calling the Api ?

Reply:
the function return POJO is ClusterData.class but set in swagger is ClusterDataImpl.class and also will show to user the incorrect return type.  It will meet some error when we through restful api to invoke the method and want to transfer the result to ClusterData.class.